File: src/main/java/com/trustline/trustline/TrustlineApplication.java
package com.trustline.trustline;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.data.jpa.repository.config.EnableJpaAuditing;

@SpringBootApplication
@EnableJpaAuditing
public class TrustlineApplication {

	public static void main(String[] args) {
		SpringApplication.run(TrustlineApplication.class, args);
	}

}

File: src/main/kotlin/trustline/TrustlineApplication.kt
package trustline

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.data.jpa.repository.config.EnableJpaAuditing

@SpringBootApplication
@EnableJpaAuditing
class TrustlineApplication

fun main(args: Array<String>) {
	runApplication<TrustlineApplication>(*args)
}
